bigchaindb_driver/connection.py

```diff
@@ -25,36 +31,79 @@ def __init__(self, *, node_url, headers=None):
         if headers:
             self.session.headers.update(headers)
 
-    def request(self, method, timeout=None, *, path=None, json=None,
-                params=None, headers=None, **kwargs):
+        self._retries = 0
+        self.backoff_time = None
+
+    def request(self, method, *, path=None, json=None,
+                params=None, headers=None, timeout=None, **kwargs):
         """Performs an HTTP requests for the specified arguments.
 
+           If the node has backoff time attached, waits till the time is due.
+
+           Backoff time is adjusted after the request is made.
```

Comment:
The description here could be a bit more detailed. When I just read it like that I ask myself where the backoff time is coming from and what it is. Next to that `backoff_time` is not directly used in here, that confused me in the beginning.